app/views/team_members/show.html.haml

```diff
@@ -32,3 +34,12 @@
             - if @team_member.instagram_link.present?
               %a{href: @team_member.instagram_link, target: "_blank"}
                 %i.fab.fa-instagram.fa-4x.text-tertiary
+
+.row.mb-2
+  .col-12.d-inline-flex.justify-content-center
+    .text-center
+      = link_to team_member_path(@team_member.prev), class: "btn btn-md border-white text-white bg-dark" do
+        <
+    .text-center
+      = link_to team_member_path(@team_member.next), class: "btn btn-md border-white text-white bg-dark" do
+        >
```

Comment:
Would would you think about adding the word "next" to this link, and use a font-awesome icon instead of just an arrow?

Reply:
Done!